pkg/exchange/bybit/bybitapi/v3/types.go

```diff
@@ -0,0 +1,15 @@
+package v3
+
+type Side string
+
+const (
+	SideBuy  Side = "0"
+	SideSell Side = "1"
```

Comment:
wondering if we can add a new type for parsing bool values like "0", "1", and 0, 1, "true", "false", true, false?

Reply:
Considering that the returned parameter is named `IsBuy`, where 0 represents Buyer instead of 1 😵‍💫, I feel it might cause confusion, so I didn't add it.

`isMaker` are the same. 0 = Maker, 1 is Taker

Reply:
Got it